etc/ansible-deploy.yaml

```diff
@@ -5,3 +5,6 @@ global_paths:
 file_naming:
   log_file_name_frmt: "ansible-deploy_execution_{}.log"
   sequence_prefix: "SEQ"
+
+permissions:
+  parent_workdir: 17407

```

Comment:
I don't think it has to be configurable, but it's fine to have it. 
As before.. never use decimental for permissions - 0o177